osmtm/models.py

```diff
@@ -512,13 +528,22 @@ def auto_fill(self, zoom):
 
     def import_from_geojson(self, input):
 
-        geoms = parse_geojson(input)
+        features = parse_geojson(input)
 
         tasks = []
-        for geom in geoms:
-            if not isinstance(geom, MultiPolygon):
-                geom = MultiPolygon([geom])
-            tasks.append(Task(None, None, None, 'SRID=4326;%s' % geom.wkt))
+        for feature in features:
+            if not isinstance(feature.geometry, MultiPolygon):
+                feature.geometry = MultiPolygon([feature.geometry])
+
+            properties = feature.properties
+
+            tasks.append(Task(
+                None,
+                None,
+                None,
+                'SRID=4326;%s' % feature.geometry.wkt,
+                properties
+            ))
```

Comment:
When we import a geoJSON file at the moment, we still lose any x, y, and z information based on L521 in the old file. So in regards to your concern on that, I don't think anything is lost with your PR. The flexibility of your `properties` implementation (instead of `import-url`) would allow someone to add a custom `x`, `y`, or `z` if they desire in the geoJSON properties.